{{ cookiecutter.__project_name_kebab }}/package.json

```diff
@@ -10,23 +10,23 @@
   "author": "",
   "license": "{% if cookiecutter.open_source_license == 'MIT license' -%}MIT{% elif cookiecutter.open_source_license == 'BSD license' %}BSD{% elif cookiecutter.open_source_license == 'ISC license' -%}ISC{% elif cookiecutter.open_source_license == 'Apache Software License 2.0' -%}Apache{% elif cookiecutter.open_source_license == 'GNU General Public License v3' -%}{% endif %}",
   "devDependencies": {
-    "@svgr/webpack": "^6.2.1",
-    "@types/react": "^16.8.19",
-    "@types/react-dom": "^16.8.19",
-    "@types/styled-components": "^5.1.23",
+    "@svgr/webpack": "^7.0.0",
+    "@types/react": "^18.2.6",
+    "@types/react-dom": "^18.2.4",
+    "@types/styled-components": "^5.1.26",
     "file-loader": "^6.2.0",
-    "postcss-loader": "^6.2.1",
-    "postcss": "^8.4.5",
-    "sass-loader": "^12.4.0",
-    "sass": "^1.45.3",
-    "ts-loader": "^9.2.6",
-    "typescript": "^4.5.5",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.9.1"
+    "postcss-loader": "^7.3.0",
+    "postcss": "^8.4.23",
+    "sass-loader": "^13.2.2",
+    "sass": "^1.62.1",
+    "ts-loader": "^9.4.2",
+    "typescript": "^5.0.4",
+    "webpack": "^5.82.0",
+    "webpack-cli": "^5.1.0"
   },
   "dependencies": {
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "styled-components": "^5.1.1"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
There are a number of places where these updates are diverging away from the major versions being used by wagtail core. For example, wagtail core still uses react 16, webpack-cli 4, etc. It feels like we should stay aligned with core on these.

@zerolab can you advise on this one?

Reply:
@lb- any thoughts on this?
My take is we're OK to upgrade some of the tooling in that core will be always more sluggish

Reply:
Ah, sorry about that. I got lazy and let PyCharm autocomplete the latest versions, but I should have followed upstream Wagtail instead.